addons/medical_treatment/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -286,6 +286,7 @@ class GVAR(actions) {
         treatmentTime = QFUNC(getStitchTime);
         condition = QFUNC(canStitch);
         callbackSuccess = "";
+        callbackStart = QFUNC(surgicalKitStart);
         callbackProgress = QFUNC(surgicalKitProgress);
         consumeItem = QGVAR(consumeSurgicalKit);
```

Comment:
not sure if needed, but might help to add a little note about how it works (not like most bools)
```suggestion
        consumeItem = QGVAR(consumeSurgicalKit); // setting can be 0,1,2 - only 1 will consume items[]
```